vmu-reporter

```diff
@@ -197,6 +195,8 @@ def generate_cell(run):
         test_log_path = glob('%soutput-%s/output/osg-test*.log' % (RUN_DIR, run['job_serial']))[0]
         test_log_filename = os.path.basename(test_log_path)
         link_location = '%s/%s' % (run_url_output_dir, test_log_filename)
+        link_text = "%s %s %s%s" % (run['tests_ok'], run['tests_ok_skip'], fail_count,
+                                    ' &#128274;' if run['selinux'] else '')
```

Comment:
Ah, ok.  FWIW, the CSL machines (mine anyway) are on Firefox 45.3.0, which apparently does not render it.